R/transformCounts.R

```diff
@@ -307,6 +346,120 @@ setMethod("transformAssay", signature = c(x = "SummarizedExperiment"),
     return(mat)
 }
 
+####################################.calc_css###################################
+# This function applies cumulative sum scaling (CSS) to the abundance table.
+.calc_css <- function(mat, css_percentile = NULL, ...) {
```

Comment:
This parameter name should be css.percentile or just percentile

Also add scaling_constant here. It should be scaling.constant or preferrably just scaling

Reply:
And if the value is 0.5 by default (as it seems to be), it should be defined here -->

css_percentile = 0.5